src/GeoBounds.cpp

```diff
@@ -1,40 +1,63 @@
-// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
 #include "GeoBounds.h"
 #include <math.h>
 
 namespace NovelRT {
-GeoBounds::GeoBounds(const NovelRT::GeoVector<float>& position, const NovelRT::GeoVector<float>& size, float rotation) : _position(position) {
+  GeoBounds::GeoBounds(const NovelRT::GeoVector<float>& position, const NovelRT::GeoVector<float>& size, float rotation) :
+    _position(position),
+    _rotation(rotation),
+    _size(size) {
+  }
 
-  _corners[0] = GeoVector<float>(position.getX() - (size.getX() / 2), position.getY() - (size.getY() / 2));
-  _corners[1] = GeoVector<float>(position.getX() + (size.getX() / 2), position.getY() - (size.getY() / 2));
-  _corners[2] = GeoVector<float>(position.getX() + (size.getX() / 2), position.getY() + (size.getY() / 2));
-  _corners[3] = GeoVector<float>(position.getX() - (size.getX() / 2), position.getY() + (size.getY() / 2));
+  bool GeoBounds::pointIsWithinBounds(const GeoVector<float>& point) const {
 
-  rotateToAngle(rotation);
-}
+    return false;
```

Comment:
Needs a todo and issue to fixup

Reply:
I'll implement this quickly now. If this isn't working then the interaction service is basically RIP in pizza AFAIK.